contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelList.vue

```diff
@@ -124,7 +124,7 @@
       },
       loadData(listType) {
         this.loading = true;
-        this.loadChannelList({ listType, sort: 'modified' }).then(() => {
```

Comment:
Is there a reason to remove this?

Reply:
Yes - adding the `sort` to the query means that it is querying indexeddb for channels with a `sort` field with the value `'modified'` - which will never match any, so it will always refetch from the server.

Any ordering from the backend order is lost after being put into the vuex state channel map - so the place to enforce the sort order of channels should be in the computed vlaue in the component, where we are already sorting by modified.

Reply:
Sounds good! Will need to figure out a different approach for when we add sort filters

Reply:
I would say that we should probably not use sort filters when we are querying a combination of indexedDB and the API, and should instead rely on client side ordering to achieve ordering operations.

For pure API calls, then we don't need to worry about this because it's not touching indexedDB.